### PR TITLE
Bug/fixing frontend

### PR DESCRIPTION
**Pull request by AbsoluteZer01**

### CODE REVIEW

File: minitwit-frontend/src/components/Registration.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
-import React from 'react';
+import React, { useSyncExternalStore } from 'react';
 import { useNavigate } from 'react-router-dom';
-
+import { useState, useEffect } from 'react';
```
Comment (ChristofferTofteng) [inline minitwit-frontend/src/components/Registration.js:3]: cant this be in the first line as well? All the things are imports from react

File: minitwit-frontend/src/components/Timeline.js
Comment (ChristofferTofteng) [inline minitwit-frontend/src/components/Timeline.js:1]: Merge the two first import lines in this file as well

File: minitwit-frontend/src/components/Registration.js (hunk 1 above)
Comment (AbsoluteZer01) [reply minitwit-frontend/src/components/Registration.js:3]: It's 2 different libaries

Comment (AbsoluteZer01) [reply minitwit-frontend/src/components/Registration.js:3]: So i would assume no, but i am not sure, gonna test it

**Pull request closed without merging**